editoast/src/views/test_app.rs

```diff
@@ -106,15 +121,7 @@ impl TestAppBuilder {
         };
 
         // Setup tracing
-        let sub = tracing_subscriber::fmt()
-            .pretty()
-            .with_env_filter(
-                tracing_subscriber::EnvFilter::builder()
-                    .with_default_directive(tracing_subscriber::filter::LevelFilter::DEBUG.into())
-                    .from_env_lossy(),
-            )
-            .with_span_events(tracing_subscriber::fmt::format::FmtSpan::CLOSE)
-            .finish();
+        let sub = create_tracing_subscriber(TracingConfig::default(), NoopSpanExporter);
```

Comment:
```suggestion
        let sub = create_tracing_subscriber(TracingConfig {
        	...
        }, NoopSpanExporter);
```

Could the default value be built here? This way we gather all default test constants here instead of having them scattered across different crates. The `impl Default` and `derive(Derivative)` from `common::tracing` can be removed.

Reply:
Done.